content/docs/reference/install.md

```diff
@@ -104,54 +108,59 @@ Subsequent updates can be installed in the usual way:
 $ brew upgrade pulumi
 ```
 
-### (Alternative) Installation Script
+### Installation Script
 
 Alternatively, our installation script can be run:
 
 ```bash
 $ curl -fsSL https://get.pulumi.com | sh
 ```
 
-This will install the `pulumi` CLI to `~/.pulumi/bin` and add it to your path.
+This will install the `pulumi` CLI to `~/.pulumi/bin` and add it to your path. When it can't automatically add `pulumi` to your path, you will be prompted to add it manually.
+See [How to permanently set $PATH on Unix](https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix) for guidance.
+
 
 The installer script can be rerun to subsequently install new updates.
 
-### (Alternative) Manual Installation
+### Manual Installation
```

Comment:
One minor nitpick I just noticed when looking over everything again:

Now that we're removing "(Alternative)" from the heading (which I agree with), should we add a sentence below all of the "Manual Installation" headings that says something like:

> Alternatively, Pulumi can be installed manually:

?

Similar to the sentence under the "Installation Script" headings?

> Alternatively, our installation script can be run: